mailbox/backup/src/test/java/org/apache/james/mailbox/backup/ZipAssert.java

```diff
@@ -127,4 +134,24 @@ public ZipAssert hasNoEntry() {
         return myself;
     }
 
+    public ZipAssert hasSameContentWith(ZipFile anotherZipFile) throws IOException {
+        if (zipFile == null || anotherZipFile == null) {
+            throwAssertionError(shouldEqual(zipFile, anotherZipFile));
+        }
+        ArrayList<ZipArchiveEntry> entries = Collections.list(zipFile.getEntries());
+        ArrayList<ZipArchiveEntry> entriesOfAnother = Collections.list(anotherZipFile.getEntries());
+
+        if (entries.size() != entriesOfAnother.size()) {
+            throwAssertionError(shouldEqual(zipFile, anotherZipFile));
```

Comment:
You should have a nice message telling that the two zip don't have the same number of entries